README.md

```diff
@@ -15,6 +15,7 @@ Beefy scenarios for Quarkus
 | 400 - 499     | QuarkConf | Quarkus scenarios presented at conferences |
 | 500 - 599     | QuarkUser | Quarkus scenarios adopted by end users |
 | 600 - 699     | QuarkSpring | Quarkus scenarios with Spring experience |
+| 700 - 799     | QuarkGrpc | Quarkus scenarios with gRPC experience |
```

Comment:
I think this should bee in `000 - 099 - Quarkus only scenarios` set

We might want to rename it to `Quarkus native scenarios`

Reply:
Ok

Reply:
Changed